client/client.go

```diff
@@ -260,12 +260,11 @@ func Ping(address string, config *Config) (bool, time.Duration) {
 }
 
 // QueryWebSocket opens a websocket connection, write `body` and return a message from the server
-func QueryWebSocket(address, body string, config *Config) (bool, []byte, error) {
+func QueryWebSocket(address, body string, config *Config, origin string) (bool, []byte, error) {
```

Comment:
```suggestion
func QueryWebSocket(address, body, origin string, config *Config) (bool, []byte, error) {
```

Can you move the origin before the config parameter? 🙏